### ISSUE TITLE
How ro add models

### ISSUE THREAD

**Issue by DannyDsds** [cocktailpeanut/fluxgym#260]
How on earth can I add models, and no just dropping it in the unet folder isn't working and yes I know I have to edit the model.yaml, but I don't know how, "training Flux Lora made easy", we can't even add models like a normal well-adjusted good boy UI.
Help.

**Comment by q1hwind** [comment]
i meet the same problem.

**Comment by rabbitbytes** [comment]
same, when i add a local model or a model from  Civitai  I add the tags into the models.yaml file .
No mater what I put into the models list it still tries to call your repo or what ever made up name i put in the repo name to huggingface.co . This causes the 404 and causes fluxgym to crash. If the model is local, why is it checking the repo for it before running.. and why default to huggingface.co anyway.

**Comment by rabbitbytes** [comment]
Ok I am on linux, my work around was place the model I want into the unet folder, i ln -s mymodelname.safetensors  flux1-dev.sft   then use the Flux-dev Model. this pulls my model and not the base flux=dev1

**Comment by vkleinmp** [comment]
Yes simply rename your desired model into "flux1-dev.sft" baking up the original helps. Should be a UI methos to easily add models.

**Comment by rabbitbytes** [comment]
or just read the folder list and use what is in there would be nice.

**Comment by vorticalbox** [comment]
you need to put the models in the follwiing folders. used ln to create symbolic link to the models i already had in comfyUI

 `flux1-dev.sft` in `models/unet`
`clip_l.safetensor` in `models/clip`
`t5xxl_fp16.safetensors` in `models/clip`
`ae.sft` in `models/vae`

what threw me was that some files use `sft` and others `safetensor` might be nice they all used `safetensor` as that is what they are called on hf etc.
